### PR TITLE
Update SaveAscii-v2.rst

### LINKED ISSUE #11810: Correct SaveAscii Documentation
This issue was originally [TRAC 10971](http://trac.mantidproject.org/mantid/ticket/10971)

SaveAscii docs still state SaveAscii does not support ragged workspace, but this has been fixed in http://trac.mantidproject.org/mantid/ticket/3493. Change documentation to reflect this.

### PR DESCRIPTION
**Pull request by lottiegreenwood**
Fixed link in docs

**To test:**

Code review

Fixes #11810.

Does not need to be in the release notes.
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [x] Has the relevant documentation been added/updated?
- [x] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by AndreiSavici** [comment]
Why not add RaggedWorkspace to concepts?

**Comment by AndreiSavici** [comment]
I would add the picture at http://www.mantidproject.org/images/e/ee/MBC_Ragged.png to the documentation, but otherwise looks good.

File: docs/source/concepts/Ragged_Workspace.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.. _Ragged_Workspace:
+
+Ragged Workspace
+=================
+
+What are they?
+--------------
+
+A ragged workspace is one where the spectra within it do not share a common set of x values. Ragged workspaces commonly occur after Converting the X units using ConvertUnits or after DiffractionFocussing.
+
```
Comment (FedeMPouzols) [inline docs/source/concepts/Ragged_Workspace.rst]: Nice to have this as a concept page!
For algorithms you can add a link to their doc page like this: 
```
:ref:`ConvertUnites <algm-ConvertUnits>`,
```
I'd suggest adding it for the algorithms mentioned here: ConvertUnits, DiffractionFocussing, Rebin (which in the old wiki page were broken links). When this is merged the old wiki page should be removed.

**Comment by AndreiSavici** [comment]
Very nice. :shipit:

**Comment by FedeMPouzols** [comment]
I noticed this PR has also added a documentation (concept) page so I'd say that should be added to the released notes, under "UI & Usability Changes" / Documentation.

**Pull request merged**